### DISCUSSION TITLE
What is the best way to fix security vulnerability in apollo-* dependecies?

### DISCUSSION THREAD

**Discussion by RishikeshDarandale** [apollographql/apollo-server#5284] [Q&A]
synk.io is [complaining](https://snyk.io/test/npm/apollo-server-core/2.25.0) about the `MEDIUM` level of security vulnerability in `apollo-server-core` which is transitively shown in case of using `apollo-server-lambda`. 

What is the best way to fix this security issue? Should I open a bug to get this fixed in `2.25.0`?

**Comment by RishikeshDarandale** [comment]
Looks like there is already [issue](https://github.com/apollographql/subscriptions-transport-ws/issues/878) exists to address this one!

**Reply by glasser** [reply]
Yes, this should be resolved either by upgrading your `ws` or your Apollo Server to 2.25.1
